fuzz/fuzz_targets/fuzz_cksum.rs

```diff
@@ -11,15 +11,16 @@ use uu_cksum::uumain;
 mod fuzz_common;
 use crate::fuzz_common::{
     compare_result, generate_and_run_uumain, generate_random_file, generate_random_string,
-    run_gnu_cmd, CommandResult,
+    pretty_print::{print_or_empty, print_test_begin},
+    replace_fuzz_binary_name, run_gnu_cmd, CommandResult,
 };
 use rand::Rng;
 use std::env::temp_dir;
 use std::fs::{self, File};
 use std::io::Write;
 use std::process::Command;
 
-static CMD_PATH: &str = "cksum";
+static CMD_PATH: &str = "../gnu/src/cksum";
```

Comment:
It means that you need gnu + build it
Currently it is using the bin from the system

Reply:
On my system, coreutils version is 9.5, so I am missing the recently added `crc32b` algorithm, which leads to many errors. IMO, it's better to fuzz against the same version of the GNU coreutils as the one we test against, but that's not a strong requirement for me, if it's too much work

Reply:
Makes sense but should be done in a different pr